packages/igx-templates/igx-ts/custom-templates/awesome-grid/files/src/app/__path__/__filePrefix__.component.ts

```diff
@@ -19,27 +19,28 @@ styleUrls: ['./<%=filePrefix%>.component.scss']
 })
 export class <%=ClassName%>Component implements OnInit, OnDestroy {
 
-  @ViewChild('grid1', { static: true, read: IgxGridComponent })
+  @ViewChild('grid1', { read: IgxGridComponent, static: true })
   public grid1: IgxGridComponent;
 
   public topSpeedSummary = CustomTopSpeedSummary;
   public bnpSummary = CustomBPMSummary;
   public localData: any[];
   public isFinished = false;
-  private isLive = true;
-  private timer;
+  private 
+  _live = true;
```

Comment:
These should be on one line. Also, rename the `private` properties using `_` to avoid lint errors in generated projects